data/hails.txt

```diff
@@ -662,6 +678,7 @@ phrase "friendly civilian"
 		"Finally dropped off some"
 		"Dropped off my cargo of"
 		"Just made a big delivery of"
+		"Recently got rid of"
```

Comment:
I mean, both are equally sentence fragments. I can remove this one if you'd like though.